src/v2/controllers/FormController.js

```diff
@@ -158,6 +173,10 @@ export default Controller.extend({
 
     // Use full page redirection if necessary
     if (model.get('useRedirect')) {
+      // Clear when navigates away from SIW page, e.g. success, IdP Authenticator
+      // Assume at those cases, SIW has done its obligation.
+      sessionStorageHelper.removeStateHandle();
```

Comment:
I'm not sure we can assume that the SIW is done (the flow can return for more remediations), but we can assume that we don't need to support page reload with current state token anymore. IF/when we return we will have a new state token.

Reply:
Yeah, I could tweak the comment. What I meant is SIW done the current transaction. When we return, widget starts a new transaction. One authentication flow could consistent multiple transactions.